[ERROR] 2025-05-02 15:18:48.383 [main] PackageRelationshipCollection - Cannot convert mailto:!@#$%^&* in a valid relationship URI-> dummy-URI used
java.net.URISyntaxException: Malformed escape pair at index 11: mailto:!@#$%^&*
	at java.net.URI$Parser.fail(URI.java:2995) ~[?:?]
	at java.net.URI$Parser.scanEscape(URI.java:3123) ~[?:?]
	at java.net.URI$Parser.scan(URI.java:3146) ~[?:?]
	at java.net.URI$Parser.checkChars(URI.java:3164) ~[?:?]
	at java.net.URI$Parser.parse(URI.java:3210) ~[?:?]
	at java.net.URI.<init>(URI.java:645) ~[?:?]
	at org.apache.poi.openxml4j.opc.PackagingURIHelper.toURI(PackagingURIHelper.java:723) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.parseRelationshipsPart(PackageRelationshipCollection.java:358) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.<init>(PackageRelationshipCollection.java:160) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.<init>(PackageRelationshipCollection.java:130) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackagePart.loadRelationships(PackagePart.java:565) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.OPCPackage.getParts(OPCPackage.java:751) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:322) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:59) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:290) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:286) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at com.Utility.ExcelReader.getSheet(ExcelReader.java:23) ~[classes/:?]
	at com.Utility.ExcelReader.getdata(ExcelReader.java:29) ~[classes/:?]
	at com.test.OLTransaction.AllScenario_OLTransaction_Cash_Transfer.gettestdate(AllScenario_OLTransaction_Cash_Transfer.java:121) ~[test-classes/:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:580) ~[?:?]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.Parameters.handleParameters(Parameters.java:809) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.Parameters.handleParameters(Parameters.java:734) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:949) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at java.util.ArrayList.forEach(ArrayList.java:1596) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:741) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestRunner.run(TestRunner.java:616) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:312) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuites(TestNG.java:1112) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.run(TestNG.java:1079) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115) [testng-remote.jar:?]
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293) [testng-remote.jar:?]
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91) [testng-remote.jar:?]
[ERROR] 2025-05-02 15:18:48.407 [main] PackageRelationshipCollection - Cannot convert mailto:!@#$% in a valid relationship URI-> dummy-URI used
java.net.URISyntaxException: Malformed escape pair at index 11: mailto:!@#$%
	at java.net.URI$Parser.fail(URI.java:2995) ~[?:?]
	at java.net.URI$Parser.scanEscape(URI.java:3123) ~[?:?]
	at java.net.URI$Parser.scan(URI.java:3146) ~[?:?]
	at java.net.URI$Parser.checkChars(URI.java:3164) ~[?:?]
	at java.net.URI$Parser.parse(URI.java:3210) ~[?:?]
	at java.net.URI.<init>(URI.java:645) ~[?:?]
	at org.apache.poi.openxml4j.opc.PackagingURIHelper.toURI(PackagingURIHelper.java:723) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.parseRelationshipsPart(PackageRelationshipCollection.java:358) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.<init>(PackageRelationshipCollection.java:160) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.<init>(PackageRelationshipCollection.java:130) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackagePart.loadRelationships(PackagePart.java:565) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.OPCPackage.getParts(OPCPackage.java:751) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:322) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:59) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:290) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:286) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at com.Utility.ExcelReader.getSheet(ExcelReader.java:23) ~[classes/:?]
	at com.Utility.ExcelReader.getdata(ExcelReader.java:29) ~[classes/:?]
	at com.test.OLTransaction.AllScenario_OLTransaction_Cash_Transfer.gettestdate(AllScenario_OLTransaction_Cash_Transfer.java:121) ~[test-classes/:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:580) ~[?:?]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.Parameters.handleParameters(Parameters.java:809) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.Parameters.handleParameters(Parameters.java:734) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:949) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at java.util.ArrayList.forEach(ArrayList.java:1596) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:741) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestRunner.run(TestRunner.java:616) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:312) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuites(TestNG.java:1112) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.run(TestNG.java:1079) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115) [testng-remote.jar:?]
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293) [testng-remote.jar:?]
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91) [testng-remote.jar:?]
[ERROR] 2025-05-02 15:18:48.409 [main] PackageRelationshipCollection - Cannot convert mailto:!@#$%^ in a valid relationship URI-> dummy-URI used
java.net.URISyntaxException: Malformed escape pair at index 11: mailto:!@#$%^
	at java.net.URI$Parser.fail(URI.java:2995) ~[?:?]
	at java.net.URI$Parser.scanEscape(URI.java:3123) ~[?:?]
	at java.net.URI$Parser.scan(URI.java:3146) ~[?:?]
	at java.net.URI$Parser.checkChars(URI.java:3164) ~[?:?]
	at java.net.URI$Parser.parse(URI.java:3210) ~[?:?]
	at java.net.URI.<init>(URI.java:645) ~[?:?]
	at org.apache.poi.openxml4j.opc.PackagingURIHelper.toURI(PackagingURIHelper.java:723) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.parseRelationshipsPart(PackageRelationshipCollection.java:358) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.<init>(PackageRelationshipCollection.java:160) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.<init>(PackageRelationshipCollection.java:130) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackagePart.loadRelationships(PackagePart.java:565) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.OPCPackage.getParts(OPCPackage.java:751) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:322) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:59) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:290) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:286) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at com.Utility.ExcelReader.getSheet(ExcelReader.java:23) ~[classes/:?]
	at com.Utility.ExcelReader.getdata(ExcelReader.java:29) ~[classes/:?]
	at com.test.OLTransaction.AllScenario_OLTransaction_Cash_Transfer.gettestdate(AllScenario_OLTransaction_Cash_Transfer.java:121) ~[test-classes/:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:580) ~[?:?]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.Parameters.handleParameters(Parameters.java:809) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.Parameters.handleParameters(Parameters.java:734) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:949) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at java.util.ArrayList.forEach(ArrayList.java:1596) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:741) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestRunner.run(TestRunner.java:616) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:312) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuites(TestNG.java:1112) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.run(TestNG.java:1079) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115) [testng-remote.jar:?]
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293) [testng-remote.jar:?]
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91) [testng-remote.jar:?]
[ERROR] 2025-05-02 15:18:48.423 [main] PackageRelationshipCollection - Cannot convert mailto:!@#$%!!! in a valid relationship URI-> dummy-URI used
java.net.URISyntaxException: Malformed escape pair at index 11: mailto:!@#$%!!!
	at java.net.URI$Parser.fail(URI.java:2995) ~[?:?]
	at java.net.URI$Parser.scanEscape(URI.java:3123) ~[?:?]
	at java.net.URI$Parser.scan(URI.java:3146) ~[?:?]
	at java.net.URI$Parser.checkChars(URI.java:3164) ~[?:?]
	at java.net.URI$Parser.parse(URI.java:3210) ~[?:?]
	at java.net.URI.<init>(URI.java:645) ~[?:?]
	at org.apache.poi.openxml4j.opc.PackagingURIHelper.toURI(PackagingURIHelper.java:723) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.parseRelationshipsPart(PackageRelationshipCollection.java:358) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.<init>(PackageRelationshipCollection.java:160) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackageRelationshipCollection.<init>(PackageRelationshipCollection.java:130) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.PackagePart.loadRelationships(PackagePart.java:565) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.OPCPackage.getParts(OPCPackage.java:751) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:322) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:59) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:290) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:286) ~[poi-ooxml-5.2.3.jar:5.2.3]
	at com.Utility.ExcelReader.getSheet(ExcelReader.java:23) ~[classes/:?]
	at com.Utility.ExcelReader.getdata(ExcelReader.java:29) ~[classes/:?]
	at com.test.OLTransaction.AllScenario_OLTransaction_Cash_Transfer.gettestdate(AllScenario_OLTransaction_Cash_Transfer.java:121) ~[test-classes/:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:580) ~[?:?]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.Parameters.handleParameters(Parameters.java:809) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.Parameters.handleParameters(Parameters.java:734) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:949) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134) ~[org.testng_7.11.0.r202503091654.jar:7.11.0]
	at java.util.ArrayList.forEach(ArrayList.java:1596) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:741) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestRunner.run(TestRunner.java:616) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:312) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.runSuites(TestNG.java:1112) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.TestNG.run(TestNG.java:1079) [org.testng_7.11.0.r202503091654.jar:7.11.0]
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115) [testng-remote.jar:?]
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293) [testng-remote.jar:?]
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91) [testng-remote.jar:?]
[INFO ] 2025-05-02 15:18:50.120 [main] Log - *** Running test method Other_Loan_Transaction_Cash_and_Transfer...
[INFO ] 2025-05-02 15:18:56.059 [main] Log - Driver has initialized successfully for firefoxbrowser
[INFO ] 2025-05-02 15:19:04.922 [main] Log - Title is displayed : BeaconPro - Login
[INFO ] 2025-05-02 15:19:07.294 [main] Log - PC Registration
[INFO ] 2025-05-02 15:19:08.430 [main] Log - Step:01 - Enter User Name in Name field.
[INFO ] 2025-05-02 15:19:09.501 [main] Log - Step:03 - Enter PC Name in Name field.
[INFO ] 2025-05-02 15:19:18.352 [main] Log - Step:03 - Select a Branch Name from the Dropdown.
[INFO ] 2025-05-02 15:19:22.668 [main] Log - Step:04 - Click on Submit Button.
[INFO ] 2025-05-02 15:19:31.751 [main] Log - Expected Result: PC Registered Successfully!
[INFO ] 2025-05-02 15:19:31.870 [main] Log - Valid Login
[INFO ] 2025-05-02 15:19:32.933 [main] Log - Step:01 - Enetered valid User Name
[INFO ] 2025-05-02 15:19:33.995 [main] Log - Step:02 - Enetered valid Password
[INFO ] 2025-05-02 15:19:58.011 [main] Log - Step:03 - Click on Login Button
[INFO ] 2025-05-02 15:19:59.079 [main] Log - Expected Result: User is logged in successfully and dashboard visible
[INFO ] 2025-05-02 15:19:59.172 [main] Log - Transactions Window Navigation
[INFO ] 2025-05-02 15:20:03.499 [main] Log - Step:01 -  Navigate through Path: Transaction: Transactions.
[INFO ] 2025-05-02 15:20:03.499 [main] Log - Expected Result:Transactions window is displayed.
[INFO ] 2025-05-02 15:20:13.243 [main] Log - Step:01 -  Navigate through Path: Transaction: Transactions.
[INFO ] 2025-05-02 15:20:13.243 [main] Log - Select Product Group and Name
[INFO ] 2025-05-02 15:20:32.918 [main] Log - Exception org.openqa.selenium.NoSuchWindowException: BrowsingContext does no longer exist
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SPCLTLAP-0350', ip: '192.168.1.6', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.6'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 138.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20250430203103, moz:geckodriverVersion: 0.36.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 35992, moz:profile: C:\Users\SANOOP~1.KUZ\AppDa..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, userAgent: Mozilla/5.0 (Windows NT 10....}
Session ID: 8b8545cb-7ef8-4ce5-b4b6-56c1d93f6c2c
*** Element info: {Using=xpath, value=.//option[@value = "OTHER LOANS"]}
